index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
```

Comment:
- [ ] Your commit messages can get better and professional. Please read [this article ](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/)to learn more about this.